### PR TITLE
update button labels

### PR DESCRIPTION
**Pull request by Olie440**

### CODE REVIEW

**Comment by Xaekai** [comment]
There is no reason to change these strings.

**Comment by Olie440** [comment]
I think it would make it a bit clearer as to what is happening.
I used what Spotify uses.

**Comment by Olie440** [comment]
@calzoneman What do you think?

**Comment by calzoneman** [comment]
I'm ok with making the button text more clear, but I'm not sure about the "Add to Playlist" label being used in this context; it could be confused with offering the ability to add the item to a saved/user playlist instead of the channel queue.

I think I would prefer if the buttons had a consistent verb, e.g. "Play next/Play last" or "Add next/Add last"; what do you think?

**Comment by Xaekai** [comment]
It would seem the consistent verb is already `queue` internally. Why not `Queue next` and `Queue last`

**Comment by Olie440** [comment]
@Xaekai Yeah sounds good!
Apologies for the way I came in, Last night I discovered the site and got excited about contributing to open source.

File: templates/channel.pug [hunk 1]
```diff
@@ -111,9 +111,9 @@ html(lang="en")
                   .input-group
                     input#mediaurl.form-control(type="text", placeholder="Media URL")
                     span.input-group-btn
-                      button#queue_next.btn.btn-default Next
+                      button#queue_next.btn.btn-default Play Next
                     span.input-group-btn
-                      button#queue_end.btn.btn-default At End
+                      button#queue_end.btn.btn-default Add to Playlist
```
Comment (calzoneman) [inline templates/channel.pug]: Looks like there is a second place that needs to be updated

Comment (Olie440) [reply templates/channel.pug]: Good spot, updated

**Comment by Olie440** [comment]
@calzoneman @Xaekai Btw if you have any tech debt you want doing let me know

**Comment by Xaekai** [comment]
Can you squash your commits?

**Comment by Olie440** [comment]
That should be a option on the merge button right?

**Comment by Olie440** [comment]
@Xaekai Thanks so much!

**Pull request merged**